Document VLM Captioning for Infographics (#1369)
NVIDIA/nv-ingest: docs/docs/extraction/vlm-embed.md

```

## VLM Captioning for Infographics Example

For documents that contain infographics (visual representations that combine text, images, charts, and diagrams), 
you can use a vision-language model (VLM) to generate descriptive captions that capture the meaning and context of the infographic.
This is particularly useful for infographics because they often contain complex visual information that benefits from natural language descriptions.

!!! note

> The VLM captioning feature requires the `vlm` profile to be enabled when starting NeMo Retriever extraction services. Refer to the [Profile Information in the Quickstart Guide](quickstart-guide.md#profile-information) for more information.

Reviewer: ```suggestion
To use VLM captioning feature, enable the `vlm` profile when you start the NeMo Retriever extraction services. For more information, refer to [Profile Information](quickstart-guide.md#profile-information).
```
